.github/workflows/release.yml

```diff
@@ -185,7 +185,7 @@ jobs:
           JIRA_DESCRIPTION: ${{ env.jira_description }}
```

Comment:
Overall, the code changes look good and seem to be aimed at updating the format of the RELEASE_NOTES variable. Here is some feedback line by line: